servlet/4.0/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta Servlet 4.0"
+date: 2019-07-24T23:21:09+0200
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+* [Jakarta Servlet 4.0 Specification Document](./servlet-4.0.pdf) (PDF)
```

Comment:
filenames are missing "-spec-" portion should be `servlet-spec-4.0.pdf` and `servlet-spec-4.0.html`

Reply:
I just clarified the checklist to say that "-spec" is preferred but not required.